core/memory_tracker/cc/posix/memory_tracker.inc

```diff
@@ -22,13 +22,18 @@ namespace track_memory {
 class PosixMemoryTracker {
   public:
     PosixMemoryTracker(std::function<bool(void*)> segfault_function):
-      signal_handler_registered_(false),
       orig_action_{0},
       handle_segfault_(segfault_function) {
     }
 
 
-  bool IsInstalled() const { return signal_handler_registered_; }
+  bool IsInstalled() const {
+    struct sigaction orig_action {
+      0
```

Comment:
Nit: does clang-format allow us to make it one line?

Reply:
I think it does if you change this to `sigaction orig_action = {0}`.